modules/nf-commons/src/main/nextflow/Const.groovy

```diff
@@ -53,17 +53,17 @@ class Const {
     /**
      * The application version
      */
-    static public final String APP_VER = "22.10.0"
+    static public final String APP_VER = "22.11.0-SNAPSHOT"
```

Comment:
```suggestion
    static public final String APP_VER = "22.10.0"
```

Reply:
haha yeah I was wondering why these files were changed! I think I was testing different parts of the build process and probably this was done with a make command I should not have run :P